### DISCUSSION TITLE
Fallback SplitHTTP

### DISCUSSION THREAD

**Discussion by dugong-lewat** [XTLS/Xray-core#3536] [General]
is splithttp fallback already possible? maybe an example like this

```
        "fallbacks": [
          {
            "path": "/splithttp",
            "dest": @sh,
            "xver": 2
          }
```

or something else?

**Comment by mmmray** [comment]
It does not support path fallbacks, please refer to https://github.com/XTLS/Xray-core/issues/3493

**Reply by dugong-lewat** [reply]
Does this mean that using a `proxy protocol` like the gRPC configuration example in [All-in-One-fallbacks-Nginx](https://github.com/XTLS/Xray-examples/tree/main/All-in-One-fallbacks-Nginx) is also not possible?

**Reply by mmmray** [reply]
that's right, grpc cannot use path fallbacks either. instead, nginx needs to be used to route grpc, and that's what the example does. you can also use nginx to route splithttp the same way.

the common issue is that both splithttp and grpc are h2 traffic, and xray fallbacks cannot read the path from h2

**Reply by dugong-lewat** [reply]
i try SplitHTTP like this its work

```
        "fallbacks": [
          {
            "alpn": "h2",,
            "dest": "1443",
          }
```
